src/components/BounceRulesContainer/Modals/CreateRuleModal/index.jsx

```diff
@@ -173,9 +157,25 @@ const CreateRuleModal = ({
                       ]}
                       onChange={handleDropdownSelect}
                       id="bounce_action"
+                      isValid={!isFieldValid(fieldValidation, "bounce_action")}
```

Comment:
isValid (as a bool) and info (as an error string) using the same naming of the function isFieldValid suggests that isFieldValid should return a bool for both cases and leads to sort of a type mismatch mentally but it actually does what it expects. How about instead of calling it isFieldValid how about make it match more what it is doing by either renaming the function to be like getFieldError or destructuring fieldValidation at the top of the function like ```cons t{ priority: priorityError, ...other stufff } = fieldValidation``` and then plugging them in as props like ```isValid={!priorityError} info={priorityError}```?